src/main/java/de/adesso/kicker/team/exception/IdenticalPlayersException.java

```diff
@@ -1,8 +1,8 @@
-package de.adesso.kicker.team;
+package de.adesso.kicker.team.exception;
```

Comment:
Don't just restructure the package structure if it's not related to you branch. This is a feature branch and this is part of refactoring. Please try to keep things separate.